logging/sd-accessing-the-service-logs.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+[id="sd-accessing-the-service-logs"]
+= Accessing the service logs for {product-title} clusters
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: sd-accessing-the-service-logs
+
+toc::[]
+
+[role="_abstract"]
+You can view the service logs for your {product-title}
+ifdef::openshift-rosa[]
+ ROSA
+endif::[]
+ clusters by using {cluster-manager-first}. The service logs detail cluster events such as load balancer quota updates and scheduled maintenance upgrades. The logs also show cluster resource changes such as the addition or deletion of users, groups, and identity providers.
+
+// Commented out while the OpenShift Cluster Manager CLI is in Developer Preview:
+//You can view the service logs for your {product-title} (ROSA) clusters by using {cluster-manager-first} or the {cluster-manager} CLI (`ocm`). The service logs detail cluster events such as load balancer quota updates and scheduled maintenance upgrades. The logs also show cluster resource changes such as the addition or deletion of users, groups, and identity providers.
+
+Additionally, you can add notification contacts for a
+ifdef::openshift-rosa[]
+ (ROSA)
+endif::[]
+ifdef::openshift-dedicated[]
+ {product-title}
```

Comment:
This reads as "a OpenShift Dedicated". I think you'll need to move the "a" or "an" into the conditionals